src/Styleguide/Components/FollowArtistPopover/index.tsx

```diff
@@ -50,14 +55,18 @@ const FollowArtistPopover: SFC<Props> = props => {
           </Box>
         </TitleContainer>
         <Flex flexDirection="column">
-          {related.suggested.edges.map(({ node: artist }, index) => {
-            return (
-              <React.Fragment key={artist.__id}>
-                <FollowArtistPopoverRow user={user} artist={artist} />
-                {index < suggetionsCount - 1 && <Separator />}
-              </React.Fragment>
-            )
-          })}
+          <Provider
+            inject={[new FollowArtistPopoverState({ excludeArtistIds })]}
+          >
+            {related.suggested.edges.map(({ node: artist }, index) => {
+              return (
+                <React.Fragment key={artist.__id}>
+                  <FollowArtistPopoverRow user={user} artist={artist} />
+                  {index < suggetionsCount - 1 && <Separator />}
+                </React.Fragment>
+              )
+            })}
+          </Provider>
         </Flex>
       </Container>
```

Comment:
Also when trying to update it end up wondering if we gain anything with adding this Container/TitleContainer just to render Box. Especially since `Container` means something in this `unstated` world.

Reply:
oh yea, ill just leave it as `Box`